packages/web/src/store/token-dashboard/sagas.ts

```diff
@@ -307,20 +307,12 @@ function* connectPhantomWallet(solana: any) {
   const disconnect = async () => {
     await solana.disconnect()
   }
-  yield connectSPLWallet(connectingWallet, solana.signMessage, disconnect)
+  yield connectSPLWallet(connectingWallet, solana, disconnect)
 }
 
-type SolanaSignMessage = (
-  encodedMessage: Uint8Array,
-  encoding: string
-) => Promise<{
-  publicKey: any
-  signature: any
-}>
-
 function* connectSPLWallet(
   connectingWallet: string,
-  solanaSignMessage: SolanaSignMessage,
+  solana: any,
```

Comment:
we got anything for this? no `Solana` type from some lib?

Reply:
It's already `any` all over the file, so maybe we can address separately